test-packages/type-tests/test/http-client.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { executeHttpRequest } from '@sap-cloud-sdk/http-client';
+import { executeHttpRequest, executeHttpRequestWithOrigin } from "@sap-cloud-sdk/http-client";
```

Comment:
[q] Lint! Why does this PR pass the check?? Are type-tests excluded from the lint checks?
```suggestion
import { executeHttpRequest, executeHttpRequestWithOrigin } from '@sap-cloud-sdk/http-client';
```

Reply:
let's enable type tests